cashu/wallet/migrations.py

```diff
@@ -43,21 +43,21 @@ async def m001_initial(db: Database):
         """
         CREATE VIEW IF NOT EXISTS balance AS
         SELECT COALESCE(SUM(s), 0) AS balance FROM (
-            SELECT SUM(amount) AS s
+            SELECT SUM(amount) 
             FROM proofs
-            WHERE amount > 0
-        );
+            WHERE amount > 0 
+        ) AS s;
     """
     )
 
     await db.execute(
         """
         CREATE VIEW IF NOT EXISTS balance_used AS
         SELECT COALESCE(SUM(s), 0) AS used FROM (
-            SELECT SUM(amount) AS s
+            SELECT SUM(amount) 
             FROM proofs_used
-            WHERE amount > 0
-        );
+            WHERE amount > 0 
+        ) AS s;
     """
     )
 
```

Comment:
I believe this is the fix and the rest is accidentally committed code?

Reply:
@callebtc  Yes, that is the fix. You can ignore the other changes for now, because those are mods to my desc_hash branch which should not be merge. 

I also hacked a fix on another view that is trying to a calculation in the SELECT field. Postgres didn't like it so I removed the second operand so the SQL statement goes through. It was for calculating the mint balance_issued-balance_redeemed. It didn't seem to be used there, so I made the change to make my stuff work.